src/open_inwoner/configurations/models.py

```diff
@@ -638,6 +638,28 @@ class SiteConfiguration(SingletonModel):
         blank=False,
         null=False,
     )
+    custom_javascript = models.FileField(
+        verbose_name=_("Custom JavaScript"),
+        upload_to="custom_scripts/",
+        blank=True,
+        null=True,
+        validators=[
+            FileExtensionValidator(allowed_extensions=["js"]),
+            validate_javascript_file,
+        ],
+        help_text=_(
+            "Upload a custom JavaScript file (.js) that is loaded in the head of every page."
```

Comment:
```suggestion
            "Upload a custom JavaScript file (.js) that will be loaded on every page."
```

(I am not sure the `<head>` part is relevant).